deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -634,24 +634,27 @@ def _yield_s3_objects(bucket: str, keys: List[str], filenames: List[str]):
         raise S3Error(f"Failed to access {bucket}/{key}: {repr(e)}")
 
 
-@app.get("/jobs/waveform_data", response_model=Union[WaveformDataResponse, GenericErrorResponse])
+@app.get("/jobs/waveform-data", response_model=Union[WaveformDataResponse, GenericErrorResponse])
```

Comment:
kind of like using kebab case for our routes instead of snake case

Reply:
same.. seemed like best practice from a few articles